dist/images/kubectl-ko

```diff
@@ -419,22 +419,36 @@ trace(){
 
   case $type in
     icmp)
+      icmp_type=""
+      if [ $af -eq 6 ]; then
+        # echo request
+        icmp_type="6.type == 128"
+      fi
       set -x
-      kubectl exec "$OVN_SB_POD" -n $KUBE_OVN_NS -c ovn-central -- ovn-trace "$ls" "inport == \"$lsp\" && ip.ttl == 255 && icmp && eth.src == $mac && ip$af.src == $srcIP && eth.dst == $dstMac && ip$af.dst == $dst && ct.new"
+      kubectl exec "$OVN_SB_POD" -n $KUBE_OVN_NS -c ovn-central -- ovn-trace --ct=new --ct=new --ct=new --ct=new "$ls" \
```

Comment:
--ct=new --ct=new --ct=new 
repeat

Reply:
```Use
 --ct=flags
    This option sets the ct_state flags that a ct_next logical
    action will report. The flags must be a comma- or space-
    separated list of connection tracking flags.
    Use multiple --ct options to specify the flags for multiple
    ct_next actions.
```

We need multiple `--ct` flags to match multiple `ct_next` actions.